tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -264,6 +264,12 @@ internal static partial class ConfigurationKeys
         /// </summary>
         public const string ApiKey = "DD_API_KEY";
 
+        /// <summary>
+        /// Configuration key for setting the default Datadog destination site, used by the Agent.
```

Comment:
The Tracer is using it now. Do we need "used by the Agent"? 
```suggestion
        /// Configuration key for setting the default Datadog destination site.
```